src/main/java/ua/edu/internship/interview/data/repository/SkillRepository.java

```diff
@@ -0,0 +1,65 @@
+package ua.edu.internship.interview.data.repository;
+
+import org.springframework.data.mongodb.repository.Aggregation;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import ua.edu.internship.interview.data.documents.SkillDocument;
+import ua.edu.internship.interview.service.dto.SkillTreeDto;
+
+import java.util.List;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.List'
<!-- __reviewdog__:ChA0N2UzYzY0YWJhYjYzYzI5EgpjaGVja3N0eWxl -->